### DISCUSSION TITLE
Review of Deepl

### DISCUSSION THREAD

**Discussion by AlexeyMatveev686** [ONLYOFFICE/onlyoffice.github.io#222] [Polls]
<img src="https://raw.githubusercontent.com/ONLYOFFICE/onlyoffice.github.io/master/discussions-icons/deepl.svg">

**Comment by Average-1** [comment]
A mess and is impossible to remove!!!

**Reply by Misokados** [reply]
Hello,
just checked the v.9.2.1, the Deepl plugin can be removed there.

**Comment by ikxncode** [comment]
Doesn't work plus it can't be removed!

**Reply by Misokados** [reply]
Hello,
just checked the v.9.2.1, the Deepl plugin can be removed there.

**Comment by ramoncunha15** [comment]
I added the API key, but it's not recognized. It doesn't work.

**Reply by Misokados** [reply]
<img width="1146" height="564" alt="image" src="https://github.com/user-attachments/assets/97ed4220-cd14-4778-9314-c5dabe770088" />
Just checked. All things are OK. Please describe the issue in details.

**Comment by berkue** [comment]
**UPDATE: THANKS A LOT!!! With version 1.0.5 the plugin works with offline files and it can also be removed, if necessary.**

I added my API key, but it always replies with "Communication failed" **and** it can't be removed!

**Reply by bischucas** [reply]
It's still not working.

**Reply by Misokados** [reply]
<img width="1146" height="564" alt="image" src="https://github.com/user-attachments/assets/71a987bc-1ad6-415c-a035-23fba94b5e49" />
Just checked API key. All things are OK. Please describe the issue in details.

**Comment by mpazbedoya-create** [comment]
Para borrarlo solo ubiquen la carpeta que contiene los plugins y simplemente bórrenlo.
